src/v2/Apps/Order/Utils/__tests__/currencyUtils.jest.ts

```diff
@@ -0,0 +1,24 @@
+import { appendCurrencySymbol } from "v2/Apps/Order/Utils/currencyUtils"
+
+describe("#appendCurrencySymbol", () => {
+  it("correctly appends US currency symbol", () => {
```

Comment:
All this control in the frontend side worries me a bit, my gut feeling tells me that we should be controlling all cases from the backend, the frontend should just display the field 

I know that was already implemented, but reading it a second time I got this feeling

Reply:
I agree with @araujobarret here! Especially given recent talks around consolidating how we display currencies across the platform, I think we should avoid having logic like this in our front-ends (it makes it harder to consolidate & maintain later).

I know that @starsirius has also been thinking about this a bit! Might be a good opportunity to pair?

Reply:
I agree with everyone as well. However, I discussed different implementation options [here](https://artsy.slack.com/archives/C9YNS4X32/p1636401545272200), and [another dev also looked into the work to see what's required](https://artsyproduct.atlassian.net/browse/NX-2405?focusedCommentId=40503) and the same conclusion was reached that handling this at the middleware or backend layer is ideal but since work on this page is coming down the Up Next pipeline, that the quickest approach to getting this done is ok for now. (Impact over perfection?) Anyways, I'm happy to revert all of this work and scope out tickets to get it done using one of the other approaches, so lemme circle back to the team and see what makes sense for next steps.

Reply:
Merged this and created a followup ticket per https://artsy.slack.com/archives/C9YNS4X32/p1638211535024400!